src/components/Sidebar/Properties/PropertyOwner.jsx

```diff
@@ -24,6 +23,111 @@ import ToggleContent from '../../common/ToggleContent/ToggleContent';
 import Property from './Property';
 import PropertyOwnerHeader from './PropertyOwnerHeader';
 
+// Render draggable and reorderable list with layers, using Beautinful DnD
+// Based on video tutorial: https://www.youtube.com/watch?v=aYZRRyukuIw&ab_channel=ColbyFayock
+function DragDropLayerList({ expansionIdentifier, uri }) {
+  const [trigger, setTrigger] = React.useState(true);
```

Comment:
The `trigger` does not seem to be used

Reply:
No, the trigger is when the "trigger" is set with the setTrigger function... it makes the component re-render